### PR TITLE
new Section::make()->aside() to align Heading & Description

### PR DESCRIPTION
**Pull request by mikrosmile**
new Section::make()->aside() to align Heading & Description on the left and content (form components inside ->schema([])) on the right

image

Upd: added breakpoints on mobile

### CODE REVIEW

**Comment by ManojKiranA** [comment]
based on #4519

File: packages/forms/resources/views/components/section.blade.php [hunk 1]
```diff
@@ -24,25 +24,28 @@
     @endif
     id="{{ $getId() }}"
     {{ $attributes->merge($getExtraAttributes())->class([
-        'filament-forms-section-component bg-white rounded-xl border border-gray-300',
-        'dark:border-gray-600 dark:bg-gray-800' => config('forms.dark_mode'),
+        'filament-forms-section-component bg-white rounded-xl border border-gray-300' => ! $isAside(),
+        'md:grid grid-cols-2 flex-row' => $isAside(),
```
Comment (zepfietje) [inline packages/forms/resources/views/components/section.blade.php]:
```suggestion
        'flex-row md:grid md:grid-cols-2' => $isAside(),
```

File: packages/forms/resources/views/components/section.blade.php [hunk 2]
```diff
@@ -91,6 +94,8 @@ class="filament-forms-section-content-wrapper"
     >
         <div @class([
             'filament-forms-section-content',
+            'bg-white rounded-xl border border-gray-300' => $isAside(),
```
Comment (zepfietje) [inline packages/forms/resources/views/components/section.blade.php]:
```suggestion
            'rounded-xl border border-gray-300 bg-white' => $isAside(),
```

File: packages/forms/resources/views/components/section.blade.php [hunk 3]
```diff
@@ -24,25 +24,28 @@
...
+        'md:grid grid-cols-2 flex-row' => $isAside(),
+        'dark:border-gray-600 dark:bg-gray-800' => config('forms.dark_mode')  && ! $isAside(),
     ]) }}
     {{ $getExtraAlpineAttributeBag() }}
 >
     <div
         @class([
             'filament-forms-section-header-wrapper flex items-center px-4 py-2 bg-gray-100 rtl:space-x-reverse overflow-hidden rounded-t-xl min-h-[40px]',
             'min-h-[56px]' => ! $isCompact(),
+            'align-top flex px-0' => $isAside(),
```
Comment (zepfietje) [inline packages/forms/resources/views/components/section.blade.php]:
```suggestion
            'flex px-0 align-top' => $isAside(),
```

File: packages/forms/resources/views/components/section.blade.php [hunk 4]
```diff
@@ -24,25 +24,28 @@
     @endif
     id="{{ $getId() }}"
     {{ $attributes->merge($getExtraAttributes())->class([
-        'filament-forms-section-component bg-white rounded-xl border border-gray-300',
-        'dark:border-gray-600 dark:bg-gray-800' => config('forms.dark_mode'),
+        'filament-forms-section-component bg-white rounded-xl border border-gray-300' => ! $isAside(),
```
Comment (zepfietje) [inline packages/forms/resources/views/components/section.blade.php]:
```suggestion
        'filament-forms-section-component rounded-xl border border-gray-300 bg-white' => ! $isAside(),
```

**Review by zepfietje — CHANGES_REQUESTED** [review]

**Comment by danharrin** [comment]
Thanks!

**Pull request merged**